tests/unit/s2n_security_policies_test.c

```diff
@@ -59,9 +62,130 @@ int main(int argc, char **argv)
             }
         }
 
-        /* If a TLS 1.3 Cipher is present in the Security Policy, then the minimum required TLS 1.3 signature algorithms
-         * must be present as well. */
         if (has_tls_13_cipher) {
+            /* Validate that s2n_tls13_default_sig_scheme() is successful on all TLS 1.3 Security Policies for all
+             * TLS 1.3 Ciphers */
```

Comment:
How are the new s2n_tls13_default_sig_scheme tests related to this change?

Reply:
`s2n_tls13_default_sig_scheme()` returning an error when an RSA Cert was configured and s2n was compiled against Openssl 1.0.2 was the runtime failure seen by the Common Runtime in this PR: https://github.com/awslabs/aws-c-io/pull/407 

In order to fully root cause the CRT's issue I first reproduced their issue in this test case. Since we previously didn't have an equivalent test for this scenario in s2n, I kept the test.